website/docs/d/database_user.html.markdown

```diff
@@ -89,7 +89,8 @@ In addition to all arguments above, the following attributes are exported:
 * `ldap_auth_type` - Method by which the provided username is authenticated. Default is `NONE`. Other valid values are: `USER`, `GROUP`.
 * `oidc_auth_type` - (Optional) Human-readable label that indicates whether the new database user authenticates with OIDC (OpenID Connect) federated authentication. If no value is given, Atlas uses the default value of `NONE`. The accepted types are:
   * `NONE` -	The user does not use OIDC federated authentication.
-  * `IDP_GROUP` - Create a OIDC federated authentication user. To learn more about OIDC federated authentication, see [Set up Workforce Identity Federation with OIDC](https://www.mongodb.com/docs/atlas/security-oidc/).
+  * `IDP_GROUP` - OIDC Workforce federated authentication user. To learn more about OIDC federated authentication, see [Set up Workforce Identity Federation with OIDC](https://www.mongodb.com/docs/atlas/security-oidc/).
```

Comment:
based on the API docs, looks like a good description could be: OIDC Workforce federated authentication group.

Reply:
Done